packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -171,7 +171,7 @@ export function useSliderThumb(
           if (e.button !== 0 || e.altKey || e.ctrlKey || e.metaKey) {
             return;
           }
-          onDown(null);
+          onDown();
```

Comment:
2nd fix: instead of moving the ColorSlider thumb into ColorSlider track element, this fixes it so mouseup properly calls onUp since the comparision between id and currentPointer.current was undefined vs null and now is undefined vs undefined.

055dd9c was the change that introduced the currentPointer stuff and indicated that it was to fix multitouch when dragging on ColorWheel/Slider, the above fix doesn't seem to cause a regression